website/src/components/Visualisations/Resources/LineGraph.tsx

```diff
@@ -44,15 +47,16 @@ class LineRechartComponent extends React.Component<IProps, any> {
         team4: 'ClientInfos.Team4.Resources',
         team5: 'ClientInfos.Team5.Resources',
         team6: 'ClientInfos.Team6.Resources',
+        CommonPool: 'CommonPool',
       },
       chartData: this.props.output,
     }
   }
 
-  handleClick = (dataKey) => {
+  handleClick = (dataKey: any) => {
     this.setState({
       disabled: this.state.disabled.includes(dataKey)
-        ? this.state.disabled.filter((obj) => obj !== dataKey)
+        ? this.state.disabled.filter((obj: any) => obj !== dataKey)
```

Comment:
same here, in general the use of `any` is frowned upon and can lead to a lot of runtime errors later on.

Reply:
Same with this one, changed to string